### ISSUE TITLE
Can we predict the data out of the sample？

### ISSUE THREAD

**Issue by dashujuzha** [PYFTS/pyFTS#34]
Dear author, can the fuzzy time series forecasting method predict future data (that is, divide the data into a training set and a test set, where the test set does not participate in building the model)? Many papers I have seen seem to only be able to train Set to fit.
